jitbuilder/release/src/Useless.cpp

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
```

Comment:
Please rename these files to something like "ControlFlowTests.[ch]pp" and please repeat the comment from the header about not being examples for how to use JitBuilder at the top of the cpp file.

After all, these really are useful tests (since it found a bona fide bug), not Useless tests :) .